crnk-gen/crnk-gen-typescript/src/test/java/io/crnk/gen/typescript/internal/TSGeneratorTest.java

```diff
@@ -57,25 +57,27 @@ public void checkMetaExcludedByDefault() {
         Assert.assertTrue(config.getExcludes().contains("resources.meta"));
     }
 
-    @Test(expected = UnsupportedOperationException.class)
-    public void throwExceptionWhenMetaElementNotMappedToNpmPackage() {
+    @Test
+    public void getNpmPackageReturnsNotSpecifiedPackageWhenMetaElementNotMappedToNpmPackage() {
         TSMetaTransformationContext transformationContext = generator.createMetaTransformationContext();
-        MetaElement metaElement = Mockito.mock(MetaElement.class);
+		MetaElement metaElement = new MetaElement();
         metaElement.setId("does.not.exist");
-        transformationContext.getNpmPackage(metaElement);
+
+		Assert.assertEquals("@packageNameNotSpecified", transformationContext.getNpmPackage(metaElement));
     }
```

Comment:
and here....

Reply:
in general I would like to chat why are we changing these tests - on the surface it looks like behaviour change but what if there are code that depends on the old behaviour?